mods/ra/weapons/missiles.yaml

```diff
@@ -75,93 +36,83 @@ Dragon:
 		ImpactSounds: splash9.aud
 		ValidImpactTypes: Water
 
+^AntiGroundMissile:
+	Inherits: ^AntiAllMissile
+	ValidTargets: Ground, Water
+
+^AntiAirMissile:
+	Inherits: ^AntiAllMissile
+	ValidTargets: Air
+
+Maverick:
+	Inherits: ^AntiGroundMissile
```

Comment:
This reintroduces the old-school sprite trails, which were intentionally removed back in the day when the line-style trails were added.